fs/glob.ts

```diff
@@ -86,9 +82,29 @@ export function isGlob(str: string): boolean {
 
 export interface ExpandGlobOptions extends GlobOptions {
   root?: string;
+  exclude?: string[];
   includeDirs?: boolean;
 }
 
+const SEP_PATTERN = isWindows ? /[\\/]+/ : /\/+/;
+
+interface SplitPath {
+  segments: string[];
+  isAbsolute: boolean;
```

Comment:
Is this property used somewhere?

Reply:
No... the paths I happen to give `split()` are necessarily absolute. I just wanted this to be a lossless path representation.